Makefile

```diff
@@ -2447,6 +2447,10 @@ endif
 .PHONY: objects
 objects: $(OBJECTS)
 
+SCALAR_SOURCES := contrib/scalar/scalar.c
```

Comment:
I would like to keep the VFS for Git part in the commit message for now, if only to give readers on the Git mailing list a chance to learn about Scalar's history (so that they gain context that this has been a multi-year effort and the design is actually quite mature).

I am totally willing to cut out that part in response to your review on the Git mailing list ;-)